README.md

```diff
@@ -22,7 +22,9 @@ Online parquet file reader demo available at:
 
 https://hyparam.github.io/hyparquet/
 
-[![hyparquet demo](demo/assets/demo.png)](https://hyparam.github.io/hyparquet/)
+[![hyparquet demo](./demo.png)](https://hyparam.github.io/hyparquet/)
```

Comment:
Once the cli pages are deployed, lets update this link to the hyparquet demo there.